morpheus/_lib/include/nvtext/subword_tokenize.hpp

```diff
@@ -22,7 +22,7 @@
 namespace nvtext {
 
 /**
- * @addtogroup nvtext_tokenize
+ * @addtogroup nvtext
```

Comment:
Does this need an end of group tag?

Reply:
I'm not sure, but end of group tag is already in there.